contracts/SimpleAuction.sol

```diff
@@ -221,6 +222,8 @@ contract SimpleAuction is /* AccessControl, */ MetaTxContext, ERC1155HolderUpgra
 
             // Transfer the NFT to the buyer.
             freeport.transferFrom(address(this), buyer, nftId, 1);
+            // Tracking amount of collateral NFTs
+            bidCollateral[seller][nftId] = 0;
```

Comment:
Why not `-1` ? I mean based on other constraints this always be 1 at this stage but that’d be more logical.